.gitignore

```diff
@@ -1,4 +1,3 @@
 node_modules/
 build/
 .direnv/
-test.js
```

Comment:
No idea why this is here, `master` doesn't have `test.js` 

Probably a github bug